src/python/pants/option/parser.py

```diff
@@ -430,8 +430,8 @@ def register(self, *args, **kwargs):
       raise FrozenRegistration(self.scope, args[0])
 
     if args:
-      dest = self.parse_dest(*args, **kwargs)
-      self._check_deprecated(dest, kwargs, print_warning=False)
+      name, dest = self.parse_name_and_dest(*args, **kwargs)
```

Comment:
Is `dest` now unused? If so, `_` or `_dest`.

Reply:
Still used.